README.md

```diff
@@ -29,18 +29,27 @@ The displayed data is real and provided by [Quandl](https://www.quandl.com). The
 
 - Download or clone this repository locally
 - Ensure [Node.js](https://nodejs.org/), which includes npm, is installed
-- Ensure [Grunt](http://gruntjs.com/getting-started#installing-the-cli) is installed:
+
+
+Install dependencies:
 
 ```
-npm install -g grunt-cli
```

Comment:
Grunt is still required for `postinstall`, when building BitFlux. (If this was removed from the Travis config, then the build would break; I think this is something which we could change in BitFlux to remove the dependency from StockFlux.)